packages/boot/src/mixins/boot.mixin.ts

```diff
@@ -49,6 +49,8 @@ import {
   ServiceOptions,
 } from '@loopback/core';
 
+/* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
I am always reluctant to disable a linter rule for the entire file, because it encourages introducing even more violations of that rule in the future.

Can you please explain why is `no-unused-vars` triggered now? Is it possible to fix the violations or at least disable the rule only for specific lines?

Reply:
Haha, you misread it. The comment is `eslint-enable` :-).

Reply:
LOL 🙈

At the moment, I see the same `eslint-enable` comment twice, can you PTAL? I think the line 28 can be removed.

https://github.com/strongloop/loopback-next/blob/b9cc1d21f44266496d2be3554023619a87824306/packages/boot/src/mixins/boot.mixin.ts#L22-L28